stargazer/APIReference/constellationRPCAPI/dag_getTransaction.md

```diff
@@ -0,0 +1,109 @@
+---
+hide_table_of_contents: true
+---
+
+<head>
+  <meta
+    name="description"
+    content="Returns information about a transaction by hash."
+  />
+</head>
+
+<intro-end />
+
+Returns information about a transaction by hash.
```

Comment:
We should make it more evident this transaction was included in a block and lives in the network.